docs/conf.py

```diff
@@ -16,6 +16,16 @@
 import importlib
 import sys
 
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    # fall back for python <= 3.7
+    # This try/catch can be removed with py <= 3.7 support
+    try:
+        import importlib_metadata
+    except ImportError:
+        pass
```

Comment:
That inner try-except isn't required because it's always installed as a dependency when needed.
```suggestion
    import importlib_metadata
```

Reply:
done